projects/plugins/jetpack/modules/blaze.php

```diff
@@ -27,7 +28,7 @@
  * @return bool
  */
 function jetpack_blaze_post_row_actions_disable( $are_quick_links_enabled, $post ) {
-	if ( 'product' !== $post->post_type ) {
+	if ( 'product' !== $post->post_type && ! ( new Host() )->is_wpcom_platform() ) {
```

Comment:
Conceptually, I think this would make more sense if it's part of a `jetpack_blaze_post_row_actions_enable` filter in `jetpack-mu-wpcom`, so we don't add custom WP.com code to the Jetpack plugin.

Reply:
Good point! Moved it in wpcomsh